tests/conftest.py

```diff
@@ -0,0 +1,93 @@
+import pytest
+from django.db import connection
+from django.db.models import Max
+from django.db.models.functions import Coalesce
+from django.utils import timezone
+
+from heroku_connect.db.models.base import HerokuConnectModel
+from heroku_connect.models import (
+    TRIGGER_LOG_ACTION, TRIGGER_LOG_STATE, TriggerLog, TriggerLogArchive
+)
+
+
+def create_trigger_log_for_model(model, *, is_archived=False, **kwargs):
+    kwargs.setdefault('table_name', model.get_heroku_connect_table_name())
+    kwargs.setdefault('record_id', model.id)
+    kwargs.setdefault('created_at', timezone.now())
+    log = make_trigger_log(is_archived=is_archived, **kwargs)
+    log.save()
```

Comment:
I don't like already saved model fixtures. They hide IO time. I want to see those expensive DB call in the test itself ;)